cumulusci/cli/cci.py

```diff
@@ -796,25 +813,53 @@ def _build_param(self, attribute, details):
         req = details["required"]
         return click.Option((f"--{attribute}",), prompt=req, required=req)
 
-    def get_command(self, ctx, name):
+    def _get_default_options(self, runtime):
+        options = []
+        options.append(
+            click.Option(
+                ("--default",),
+                is_flag=True,
+                help="Set this service as the global defualt.",
+            )
+        )
+        if runtime.project_config is not None:
+            options.append(
+                click.Option(
+                    ("--project",),
+                    is_flag=True,
+                    help="Set this service as the default for this project only.",
+                )
+            )
+        return options
+
+    def get_command(self, ctx, service_type):
         runtime = RUNTIME
         runtime._load_keychain()
         services = self._get_services_config(RUNTIME)
+
         try:
-            service_config = services[name]
+            service_config = services[service_type]
         except KeyError:
-            raise click.UsageError(f"Sorry, I don't know about the '{name}' service.")
-        attributes = service_config["attributes"].items()
+            raise click.UsageError(
+                f"Sorry, I don't know about the '{service_type}' service."
+            )
 
+        attributes = service_config["attributes"].items()
         params = [self._build_param(attr, cnfg) for attr, cnfg in attributes]
-        if runtime.project_config is not None:
-            params.append(click.Option(("--project",), is_flag=True))
+        params.extend(self._get_default_options(runtime))
 
         def callback(*args, **kwargs):
             if runtime.project_config is None:
-                project = False
+                set_project_default = False
             else:
-                project = kwargs.pop("project", False)
+                set_project_default = kwargs.pop("project", False)
+
+            set_global_default = kwargs.pop("default", False)
+            if set_global_default and set_project_default:
+                raise click.UsageError(
+                    "Cannot specify both --default and --project. Please choose one."
```

Comment:
This is fine, but there's no technical reason they need to be mutually exclusive.

Reply:
I switched it to accept both.